doc/user/QUICKSTART.md

```diff
@@ -0,0 +1,150 @@
+# Quickstart Guide
+
+This guide will help you get started using F3D.
+
+As an overview, you'll learn how to run F3D and open supported files. You'll then learn how to configure basic scene constructions, animations and plugins.
+
+## Prerequisites
+
+To use F3D, you'll need to install F3D. See the [Installation](INSTALLATION.md) page for the latest release package supported by your system.
+
+## Running F3D
+
+Once F3D has been installed, you should be able to open a [supported file](#supported-file-formats) in any of the following ways:
+
+* Open a file directly from your file manager.
+* Open F3D, then drag and drop a file into the application window.
+* Run F3D from the terminal with a set of command-line [options](OPTIONS.md).
+* Configure F3D as a [thumbnailer](DESKTOP_INTEGRATION.md) for supported file formats with certain file managers.
+
+If you choose to run F3D from the terminal, you have different options.
+
+1\. Open a file and visualize it interactively:
+
+```
+f3d /path/to/file.ext
+```
+
+2\. Open a file and save the rendering into an image file:
+
+```
+f3d /path/to/file.ext --output=/path/to/img.png
+```
+
+If you need help, specify the `--help` [option](OPTIONS.md):
+
+```
+f3d --help
+man f3d # Linux only
+```
+
+## Supported file formats
+
+F3D supports the following file formats:
```

Comment:
should we still list formats here ? if we do it should be worded differently imo

Reply:
removed it, adding a note linking to full list at the end of the "running F3D" section